components/RecentStories.vue

```diff
@@ -4,38 +4,46 @@
       v-for="(story, index) in recentStories"
       :key="index"
       class="gothamist mod-small"
-      :image="story.lead_asset[0] && story.lead_asset[0].value.default_image ? story.lead_asset[0].value.default_image.file : defaultImage"
+      :image="story.lead_asset[0] && story.lead_asset[0].value.image ? story.lead_asset[0].value.image.file : defaultImage"
       :image-height="150"
       :image-width="150"
+      :sponsored="story.sponsored_content"
       :title="story.title"
       :title-link="story.url"
       :tags="story.tags"
     >
-      <!--      <article-metadata-->
-      <!--        :publish-date="story.meta.first_published_at"-->
-      <!--      >-->
-      <!--        <template v-slot:comments>-->
-      <!--          <v-counter-->
-      <!--            icon="comment"-->
-      <!--            value="40"-->
-      <!--            href="http://www.google.com"-->
-      <!--          />-->
-      <!--        </template>-->
-      <!--      </article-metadata>-->
+      <article-metadata
+        :publish-date="story.updated_date ? null : fuzzyDateTime(story.meta.first_published_at)"
+        :updated-date="story.updated_date ? fuzzyDateTime(story.updated_date) : null"
+      >
+        <template
+          v-if="disqusData && disqusData.data.response[index]"
+          v-slot:comments
+        >
+          <v-counter
+            icon="comment"
+            :value="disqusData.data.response[index].posts"
+            :href="story.url + '?to=comments'"
+          />
+        </template>
+      </article-metadata>
     </v-card>
   </div>
 </template>
 
 <script>
 import { mapState } from 'vuex'
+import disqus from '~/mixins/disqus'
+import helpers from '~/mixins/helpers'
 
 export default {
   name: 'RecentStories',
   components: {
-    // ArticleMetadata: () => import('nypr-design-system-vue/src/components/ArticleMetadata'),
-    // VCounter: () => import('nypr-design-system-vue/src/components/VCounter'),
+    ArticleMetadata: () => import('nypr-design-system-vue/src/components/ArticleMetadata'),
+    VCounter: () => import('nypr-design-system-vue/src/components/VCounter'),
     VCard: () => import('nypr-design-system-vue/src/components/VCard')
   },
+  mixins: [disqus, helpers],
```

Comment:
I'm still not really a fan of mixins for standalone helper functions that can just be plain imports. Can we do something like `import { getCommentCount } from '../utils/disqus'` (or wherever we put it).  It's nice to be able to clearly see the connection from the function import to the function instead of guessing that the `this.getCommentCount` farther down here is probably? coming from the `disqus` mixin. For the date helper too.

I can understand the case for mixins that need access to `this` from the component, but even in those cases we should probably be moving toward the composition API. https://v3.vuejs.org/guide/mixins.html#precautions.

Reply:
I changed the date helper to an import as we discussed, left the disqus as a mixin for now since I need `this` and access to the state. When we upgrade to Nuxt 3 I'd love to try out the composition API more though!